scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -414,6 +414,14 @@ class FormatOps(
     template.tokens.find(x => x.is[T.LeftBrace] && owners(x) == template)
   }
 
+  def templateWith(template: Template): Option[Token] = {
```

Comment:
1. i think that method `templateCurly` on line 409 is unused, i'd remove it
2. this method is used in only one place; should we inline it there?

Reply:
1. Removed.
2. Inlined.
:tada: